packages/sanity/src/core/releases/tool/detail/ReleaseDetail.tsx

```diff
@@ -18,10 +17,10 @@ import {ReleaseReview} from './ReleaseReview'
 const SUPPORTED_SCREENS = ['overview', 'review'] as const
 type Screen = (typeof SUPPORTED_SCREENS)[number]
 
-const useFetchBundleDocuments = (bundleName: string) => {
+const useFetchBundleDocuments = (bundleSlug: string) => {
   const client = useClient({apiVersion: API_VERSION})
-  const query = `*[defined(_version) &&  _id in path("${bundleName}.*")]`
-  return useListener<BundleDocument>({query, client})
+  const query = `*[defined(_version) &&  _id in path("${bundleSlug}.*")]`
+  return useListener({query, client})
```

Comment:
Just wondering: is it intentional to remove the listener type argument (`BundleDocument`) altogether?

Reply:
It was a mistake to include this as `BundleDocument`, it is a version document, so it's any type of documents. 
Given it was extending `SanityDocument` ts was not warning us

Reply:
Ah of course! These are version documents, not bundle metadata documents. My bad!